autogpt_platform/backend/backend/copilot/graphiti/falkordb_driver_test.py

```
@@ -296,3 +303,160 @@ async def test_execute_query_already_indexed_returns_none_without_retry(
+
+# --- guards for the fulltext index-creation writes -------------------------
+
+
+@pytest.mark.parametrize(
+    "cypher",
+    [
+        # graphiti-core builds all three fulltext indices this way. `\bCREATE\b`
+        # does NOT match `createNodeIndex`, so these were classified read-only
+        # and bounced off RO_QUERY on every new group's first write.
+        "CALL db.idx.fulltext.createNodeIndex({label: 'Episodic', stopwords: []},"
+        " 'content', 'source', 'source_description', 'group_id')",
```

Ruff (0.16.2)

[warning] Unparenthesized implicit string concatenation in collection

Did you forget a comma?

(ISC004)